app/views/admin/vouchers/new.html.haml

```diff
@@ -1,22 +1,23 @@
-.row
-  .sixteen.columns.alpha
-    .four.columns.alpha.text-right
-      %a.button{ href: "#{edit_admin_enterprise_path(@enterprise)}#!#vouchers_panel"}
-        = t('.back')
-    .twelve.columns.omega
-      .row 
-        .eight.columns.text-center
-          %legend= t(".legend")
-        .four.columns.text-right
-          %input.red{ type: "button", value: t('.save') }
-      .row
-        .alpha.four.columns
-          = label :voucher, :code, t('.voucher_code')
-        .omega.eight.columns
-          %textarea.fullwidth{ id: 'voucher_code', name: 'voucher_code', rows: 6 }
-      .row
-        .alpha.four.columns
-          = label :voucher, :amount, t('.voucher_amount')
-        .omega.eight.columns
-          $
-          %input{ type: 'text', id: 'voucher_amount', name: 'voucher_amount', value: 10, disabled: true }
+= form_with model: @voucher, url: admin_enterprise_vouchers_path(@enterprise), html: { name: "voucher_form" } do |f|
```

Comment:
Why does it have a name?

Reply:
No particular reason, I must copy pasted the `form_with` from somewhere else.